### DISCUSSION TITLE
Consider dropping "CBL" from the name

### DISCUSSION THREAD

**Discussion by dylanmtaylor** [microsoft/azurelinux#6328] [General] [answered]
CBL-Mariner isn't short and easy to say, so I imagine most people just say "Mariner" when referring to this project. "Microsoft Mariner" is an awesome name, and if it's necessary to associate the project with Linux or something, "Mariner Linux" is more descriptive than "CBL-Mariner". Just a thought.

**Comment by mfrw** [comment, marked as answer]
For the 3.0 release it proposed to be called `Azure Linux`
- https://github.com/microsoft/CBL-Mariner/pull/6275
